R/count_watershed_data.R

```diff
@@ -221,9 +221,30 @@ count_watershed_data <- function(data_dir,
 
         #-------------------------------------------------------
         # TELECONNECTION - WASTE WATER TREATMENT PLANTS
-        wasteflow_points[watersheds_select, ] -> wasteflow_select
+        wasteflow_points[watersheds_select, ] %>%
+          as.data.frame() %>%
+          dplyr::select(c("FACILITY_NAME",
+                          "CWNS_NUMBER",
+                          "PRIMARY_WATERSHED_HUC",
+                          "PRIMARY_WATERSHED_NAME",
+                          "lon",
+                          "lat",
+                          "DISCHARGE_METHOD",
+                          "EXIST_TOTAL",
+                          "PROJ_TOTAL",
+                          "PRES_RES_RECEIVING_COLLCTN",
+                          "PRES_RES_ONSITE_WTS"
+          )) -> wasteflow_select
 
+        wasteflow_select[is.na(wasteflow_select)] <- 0
 
+        sum(wasteflow_select$EXIST_TOTAL) -> totalflow_thru_MGD
+
+        sum(wasteflow_select$PRES_RES_RECEIVING_COLLCTN) -> total_pop_served
```

Comment:
We want to make sure that the population served is calculated only for rows that have a discharge value. So I suggest something like:
```
wasteflow_select %>%
   filter(!is.na(EXIST_TOTAL) & !is.na(PRES_RES_RECEIVING_COLLCTN)) -> wasteflow_select_complete

wasteflow_select_complete["EXIST_TOTAL"] %>% sum() -> totalflow_thru_MGD
wasteflow_select_complete["PRES_RES_RECEIVING_COLLCTN"] %>% sum() -> total_pop_served
```